pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "mteb"
-version = "1.38.1"
+version = "1.38.2"
```

Can you roll back pyproject?

```suggestion
version = "1.38.1"
```

---

Done